From 4617b97941d56d655591eec5af9e1eb94a16aa4b Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 18 Nov 2016 16:49:49 +0900
Subject: [PATCH 1539/2066] arm64: dts: r8a7795-salvator-x: Increase
 spi-max-frequency to 66.66 MHz for MSIOF ch1

The maximum clock of the MSIOF ch1 module is 66.66 MHz from R8A7795
ES2.0 SoC. This patch changes for that.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
index 9e9b313..650be53 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x.dts
@@ -959,7 +959,7 @@
 	spidev@0 {
 		compatible = "renesas,sh-msiof";
 		reg = <0>;
-		spi-max-frequency = <33333333>;
+		spi-max-frequency = <66666666>;
 		spi-cpha;
 		spi-cpol;
 	};
-- 
1.7.5.4

